src/components/SignUp.jsx

```diff
@@ -97,26 +103,38 @@ function SignUp() {
     setRole("role");
   };
 
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    alert("Account created!");
-    clearForm();
-  };
-
   const handleFormSubmit = (e) => {
     e.preventDefault();
-    if (getIsFormValid()) {
-      handleSubmit(e);
-    }
+    const data = {
+      fullName: fullName.value,
+      email: email.value,
+      mobileNumber: mobileNumber.value,
+      username: userName.value,
+      password: password.value,
+      role: role,
+    };
+    
+    axios
+      .post("https://15f6-203-212-221-36.ngrok-free.app/api/users/sign-up", data) 
```

Comment:
use .env file to store the API endpoints

Reply:
changes done